mf2py/parser.py

```diff
@@ -347,21 +349,18 @@ def parse_props(el):
             # if this is not a property element, but it is a h-* microformat,
             # add it to our list of children
             if not is_property_el and root_class_names:
```

Comment:
shouldn't this add `h` to the `parsed_type_aggregation`?

Reply:
This should be covered in [Line 258](https://github.com/microformats/mf2py/pull/119/files/bb4b424987b057fb67f7f509278dbe1fd3fd8b14#diff-5b41ecb01a373b615df9696a73cc84afR258). Since it doesn't matter if it is a child or a nested property, I add it there already. Unless I missed a code path?

Reply:
aah yes I missed that change